src/pkgmt/config.py

```diff
@@ -26,18 +30,71 @@ def __repr__(self) -> str:
         return f"{type(self).__name__}({self._data!r})"
 
 
-def load():
-    if Path("pyproject.toml").exists():
-        try:
-            with open("pyproject.toml") as f:
-                return Config(toml.load(f)["tool"]["pkgmt"], "pyproject.toml")
-        except toml.decoder.TomlDecodeError as e:
-            raise ValueError(
-                f"Invalid pyproject.toml file: {str(e)}."
-                "If using a boolean "
-                "value ensure it's in lowercase, e.g., key = true"
+class PyprojectConfig:
```

Comment:
this class isn't adding much value on its own, it'd be better for merge it with Config, you can make PyprojectConfig.__init__ a class method in Config

```python
cfg = Config.from_config_file("pyproject.yoml")
```

Reply:
Merged with Config